core-bundle/src/Resources/contao/drivers/DC_Table.php

```diff
@@ -4811,6 +4848,8 @@ protected function listView()
 
 			foreach ($result as $row)
 			{
+				$this->denyAccessUnlessGranted(ContaoCorePermissions::DCA_VIEW, new DataContainerSubject($this->strTable, $row['id']));
```

Comment:
this means if I want to limit an element from being visible, I must not use this permission but also apply a `filter` or similar to the DCA, right? Otherwise we should catch the exception and not list the item.

Reply:
Yes, this is only a security measure here.